src/app/themes.tsx

```diff
@@ -30,7 +29,7 @@ export type rainbowPalette = {
 
 export type customTheme = {
     light: boolean;
-    main: standardPalette & { border: string };
+    main: standardPalette & { lighterForeground: string; border: string };
```

Comment:
Do we need a `lighterForeground`? Just seems like we already have a lot of colours going on

Reply:
Yes we do need
lighterForeground is specifically used for less important text - e.g. the forget your password line on the login panel
Also two main color is needed (it replaced the surfaceForegroundColor initially in the older themes) - use is also demonstrated in login panel